webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -226,7 +277,12 @@ describe('ContributionForm.vue', () => {
             content: 'auf Deutsch geschrieben',
             language: 'de',
             image,
-            categories: [{ id: 'cat12', name: 'Democracy & Politics' }],
+            categories: [
+              {
+                id: 'cat12',
+                name: 'Democracy & Politics',
+              },
+            ],
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-09T20:20:26Z" title="Tuesday, July 9th 2019, 10:20:26 pm +02:00">Jul 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -201,7 +220,12 @@ describe('ContributionForm.vue', () => {
             content: 'auf Deutsch geschrieben',
             language: 'de',
             image,
-            categories: [{ id: 'cat12', name: 'Democracy & Politics' }],
+            categories: [
+              {
+                id: 'cat12',
+                name: 'Democracy & Politics',
+              },
+            ],
```


@ogerly try not to touch unrelated files. First, it makes revieweing your PR difficult. Second and more important, you will "own" the code. This will make it difficult to understand the code, because `git blame` will tell me that you are the author, but you just changed indentation.

To fix this: Run`git checkout origin/master -- path/to/unrelated/files`.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-07-10T04:53:44Z" title="Wednesday, July 10th 2019, 6:53:44 am +02:00">Jul 10, 2019</time>_

---


yes, i'm sorry. 
i have a lint problem in my editor. 
when i edit a page that i later restore to its original state. there is still a change in the git status due to the lint. 
should i just turn it off in my editor ? 
![Bildschirmfoto vom 2019-07-10 06-51-51](https://user-images.githubusercontent.com/1324583/60941430-6d7fa780-a2df-11e9-856d-492757060dc5.png)